scripts/build_lilypond.sh

```diff
@@ -42,8 +53,9 @@ mkdir -p "$LILYPOND_BUILD"
     GHOSTSCRIPT="$GHOSTSCRIPT_INSTALL/bin/gs" \
     GUILE="$GUILE_INSTALL/bin/guile" GUILE_CONFIG="$GUILE_INSTALL/bin/guile-config" \
     PYTHON="$PYTHON_INSTALL/bin/python" PYTHON_CONFIG="$PYTHON_INSTALL/bin/python-config" \
+
     "$LILYPOND_SRC/configure" --prefix="$LILYPOND_INSTALL" --disable-documentation \
         --enable-static-gxx --enable-relocation
-    $MAKE -j$PROCS
-    $MAKE install
-) > "$LILYPOND/build.log" 2>&1
+
+    $MAKE -j$PROCS && $MAKE install
+) > "$LILYPOND/build.log" 2>&1 || print_failed "$LILYPOND_BUILD/build.log"
```

Comment:
I see the benefit of `print_failed` and like it, but does it still need to fuse all calls to `make` into one pipe? I looked it up and POSIX says `set -e` propagates to subshells.

Reply:
Hm, yes this is weird. `set -e` should have taken care of this. I chained `make` and `make install` to prevent the latter from being run if `make` failed, but, as you say, `set -e` should've been enough. I'll have to investigate this further.